content/projects/web-development-website/_index.md

```diff
@@ -0,0 +1,64 @@
+---
+content_type: project
+flavours:
+- none
+learning_outcomes:
+- web_dev_hosting
+- web_dev_responsive_page_design
+- web_dev_web_page_structure
+- web_dev_web_design_principals
+prerequisites:
+  hard:
+  - projects/simple-website
+ready: true
+story_points: 3
+submission_type: link
+tags:
+- html
+- css
+- responsive web design
+title: Web Development - Build your first personal website
+---
+
+Now that you have finished creating your first simple personal portfolio, make it responsive and ensure that it passes through a linter. You should make changes to the same portfolio you have created, do not make a new one.
+
+## Instructions
+
+The HTML and CSS should be developed in a logical way with good naming conventions that would allow another developer to easily understand and build upon your code. There is something called "Semantic Markup" - read all about it [here](https://seekbrevity.com/semantic-markup-important-web-design/).
+
+You can also make use of this cool [markup validation tool](https://validator.w3.org/) to validate the HTML code used in your website.
+
+Your web page files and folders should be organized and named in a logical manner.
+
+Once you have finished your design and are satisfied with the style and content, please submit a link of the website hosted on GitHub Pages. You must have hosted your simple portfolio to [Github Pages](https://guides.github.com/features/pages/) by now, please submit the same link of your website with the updated features. DO NOT submit a link to your github repo. We don't want to clone your website, we want to look at what you built.
+
+### Requirements:
```

Comment:
The headings are inconsistent. Are they all meant to have colons? Or are they all meant to not have colons?